package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "react-date-range",
-  "version": "1.0.0-beta",
+  "name": "@researchgate/react-date-range",
```

Comment:
what's the case here?

Reply:
Oops, that's our versioned release. I can correct that.

Reply:
@keremciu Done. I set the name back.